File: LeetCode/Python/Two_Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(0,len(nums)):
            target_number = target-nums[i]
            if(target_number in nums):
                for j in range(0,len(nums)):
                    if(nums[j]==target_number) and (i!=j):
                        return [i,j]
            else:
                continue

File: LeetCode/C++/TwoSum.cpp
#include <map>
using namespace std;

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> indices;
        map<int,int> has_sum;
        int desired_value; 
        for(int i =0;i<=nums.size();i++)
        {
            desired_value = target-nums[i];
            if(has_sum.count(desired_value))
            {
                indices.push_back(i);
                indices.push_back(has_sum[desired_value]);
                return indices;
            }
            else
            {
                has_sum[nums[i]]=i;
            }
        }
    return indices;
    }
};
